flycheck.el

```diff
@@ -3610,6 +3663,40 @@ list."
       ;; highlighting
       (flycheck-error-list-highlight-errors preserve-pos))))
 
+(defun flycheck-error-list-mode-line-filter-indicator ()
+  "Create a string representing the current error list filter."
+  (if flycheck-error-list-minimum-level
+      (format " [>= %s]" (symbol-name flycheck-error-list-minimum-level))
+    ""))
+
+(defun flycheck-error-list-set-filter (level)
+  "Restrict the error list to errors at level LEVEL or higher."
+  (interactive
+   (list (read-flycheck-error-level
+          "Minimum error level (errors at lower levels will be hidden): ")))
+  (-when-let (buf (get-buffer flycheck-error-list-buffer))
+    (with-current-buffer buf
+      (set (make-local-variable 'flycheck-error-list-minimum-level)
```

Comment:
Done! Do all emacs version that we support have it? I saw that the rest of flycheck used `(set (make-local-variable ...) ...)`, so I initially did the same.